trufflehog/AbstractPathfinder.cpp

```diff
@@ -0,0 +1,11 @@
+/*
+Author: Isha Dijcks <i.e.dijcks@student.tudelft.nl>
+*/
+
+#include "AbstractPathfinder.h"
+
+#ifdef DEBUG
+void TruffleHog::AbstractPathfinder::set_verbose(const bool on) {
+    verbose = on;
+}
+#endif
```

Comment:
```suggestion
#endif

```
Add newline at end of file